pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -757,6 +781,92 @@ func etTypeConverter(et binding.ExtendedType, v string) string {
 	}
 }
 
+func goTypeConverter(et binding.ExtendedType, v string) string {
+	switch et.Base {
+	case smartcontract.AnyType:
+		return "stackitem.TryMake(" + v + ")"
+	case smartcontract.BoolType:
+		return "stackitem.NewBool(" + v + "), error(nil)"
+	case smartcontract.IntegerType:
+		return "(*stackitem.BigInteger)(" + v + "), error(nil)"
```

Comment:
Are you sure we don't need `stackitem.NewBigInteger`? It implies additional size check.